src/main/webapp/app/entities/event-activity/event-activity-create.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { Button, Row, Col, Label } from 'reactstrap';
+import { AvForm, AvGroup, AvInput, AvField } from 'availity-reactstrap-validation';
+import { Translate } from 'react-jhipster';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IRootState } from 'app/shared/reducers';
+import { createEntity, reset } from './event-activity.reducer';
+import { getEntity as getEventEntity } from '../event/event.reducer';
+import { convertDateTimeToServer } from 'app/shared/util/date-utils';
+import moment from 'moment';
+import { APP_LOCAL_DATETIME_FORMAT } from 'app/config/constants';
+
+export interface IEventActivityCreateProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string; eventId: string }> {}
+
+export interface IEventActivityCreateState {
+  isNew: boolean;
+}
+
+export class EventActivityCreate extends React.Component<IEventActivityCreateProps, IEventActivityCreateState> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isNew: !this.props.match.params || !this.props.match.params.id
```

Comment:
maybe this is cleaner?
```
!this.props.match?.params?.id
```

Reply:
i think i will remove this
since it is not using anymore 😆

Reply:
resolved in 915d6223536e7bce48768a8e19037bd9a1cb9765